supervisor/supervisor.mk

```diff
@@ -94,9 +94,40 @@ ifndef USB_HID_DEVICES
 USB_HID_DEVICES = "KEYBOARD,MOUSE,CONSUMER,GAMEPAD"
 endif
 
-# SAMD21 needs separate endpoint pairs for MSC BULK IN and BULK OUT, otherwise it's erratic.
-ifndef USB_MSC_NUM_ENDPOINT_PAIRS
-USB_MSC_NUM_ENDPOINT_PAIRS = 1
+ifndef USB_RELATIVE_EP_NUM
```

Comment:
I'd call this `USB_RENUMBER_ENDPOINTS` and if it's 0 for off then make sure the descriptor generator errors out if an endpoint number it uses is not provided. (I mention this on the other PR too.)

Reply:
Done!